data/data/aws/route53/base.tf

```diff
@@ -85,6 +85,14 @@ resource "aws_route53_record" "etcd_a_nodes" {
   records = [var.etcd_ip_addresses[count.index]]
 }
 
+resource "aws_route53_record" "bootstrap_a_node" {
```

Comment:
nit, we try to keep all the bootstrap resources ie resources that are used by bootstrap-host and should be deleted when bootstrapping is complete, in the bootstrap module.

So I think a better way would be to pass the internal_zone_id, cluster_domain to the bootstrap module and create the dns record there??

Reply:
We would like to resolve this in a patch after this PR if possible.

Reply:
> We would like to resolve this in a patch after this PR if possible.

hmm, this shouldn't be too much change.. 
just want to make sure we don't delete resources that aren't meant to be deleted..

Reply:
agreed